cinema/views.py

```diff
@@ -35,7 +36,7 @@ class GenreViewSet(
 ):
     queryset = Genre.objects.all()
     serializer_class = GenreSerializer
-    authentication_classes = (TokenAuthentication,)
+    # authentication_classes = (TokenAuthentication,)
```

Comment:
The `authentication_classes` line is commented out in the `GenreViewSet`. If authentication is intended to be used, ensure this line is uncommented and properly configured.